features/calculation_weekly_frequency.feature

```diff
@@ -0,0 +1,26 @@
+Feature:
+  As a enrolee into a Workplace Pension,
+  I would like to calculate pension contribution on my weekly salary.
+
+  Background:
+    Given I am on the YourDetailsPage
+
+    Scenario Outline:
+      Given I enter my age as "<age>"
```

Comment:
@willhall88 let's pair on a solution (or to leave as it is) to our cucumber steps in a separate PR, ok?